src/config.ts

```diff
@@ -324,17 +323,6 @@ export const BONUS_API_CLIENT = BonusAPIClient(
   BONUS_API_URL,
   httpApiFetch
 );
-// the date until a user can request a bonus
-export const BONUS_REQUEST_LIMIT_DATE = UTCISODateFromString.decode(
-  process.env.BONUS_REQUEST_LIMIT_DATE
```

Comment:
Reference to `BONUS_REQUEST_LIMIT_DATE` should be removed from README end `.env.example`